### PR TITLE
Changed breadcrumbs styles

### PR DESCRIPTION
**Pull request by bexsoft**
## What does this do?
- Changed breadcrumbs styles
- Added onKey press support to input boxes
- Added enter key support to create path modal
- Replaced bucket icon

## How does it look?
Screen Shot 2022-02-21 at 18 43 05
Screen Shot 2022-02-21 at 18 42 38
Screen Shot 2022-02-21 at 18 41 17
Screen Shot 2022-02-21 at 18 41 10

Signed-off-by: Benjamin Perez <benjamin@bexsoft.net>

### CODE REVIEW

**Review by dvaldivia — APPROVED** [review]

File: portal-ui/src/screens/Console/Common/FormComponents/common/styleLibrary.ts [hunk 1]
```diff
@@ -388,6 +395,17 @@ export const objectBrowserCommon = {
...
+  breadcrumbsList: {
+    textOverflow: "ellipsis" as const,
+    overflow: "hidden" as const,
+    whiteSpace: "nowrap" as const,
+    display: "inline-block" as const,
+    flexGrow: 1,
+    textAlign: "left" as const,
+    marginLeft: 15,
+    marginRight: 10,
+    lineHeight: 35,
```
Comment (prakashsvmx) [inline portal-ui/src/screens/Console/Common/FormComponents/common/styleLibrary.ts:407]: this could be removed.

```suggestion
```

**Comment by prakashsvmx** [comment]
The Create new path icon could be changed : @oscarocastellanos 
image

**Review by prakashsvmx — APPROVED** [review]
changes look good to me. :+1:

**Comment by oscarocastellanos** [comment]
@prakashsvmx @bexsoft This is the icon we use for Create new path 👍. Thanks
[Create New Path Icon.svg.zip](https://github.com/minio/console/files/8118399/Create.New.Path.Icon.svg.zip)
!

**Comment by bexsoft** [comment]
Will apply changes on the next PR

**Pull request merged**